hummingbird/ml/_topology.py

```diff
@@ -55,17 +55,32 @@ def _jit_model(torch_model, trace_input, device, extra_config):
     return torch.jit.trace(torch_model, trace_input).eval()
 
 
-def _get_trace_input_from_test_input(input, remainder_size=None):
+def _get_trace_input_from_test_input(input, remainder_size=None, extra_config={}):
     """
     Utility function used to properly put the inputs into a format understandable by torch.
     If `remainder_size` is provided, also return inputs for a remainder model (see below).
     """
     remainder = None
     if isinstance(input, tuple):
-        trace_input = tuple([torch.from_numpy(i) for i in input])
+        trace_input = []
+        for input_ in input:
+            # Convert string arrays into int32.
+            if input_.dtype.kind in {"U", "S"}:
```

Comment:
I see "{u, s}" in several places. Should we make a constant-ish thing? (I doubt it will change but just a thought)

Reply:
I can add something to constants maybe. Let me try.